config/locales/en.yml

```diff
@@ -629,6 +630,11 @@ en:
       media_not_ready: Audio not ready
       notice: Episode updated
   episode_media:
+      title:
+        enclosure_override: Enclosure Override
+      help:
+        enclosure_override_url: UNIQUE HELP TEXT
+        enclosure_override_prefix: UNIQUE HELP TEXT
```

Comment:
@kookster I wasn't sure how to describe these fields in a helpful way, so I added a placeholder for now.